app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -73,6 +87,8 @@ public class MainActivity : AppCompatActivity(), ViewController,
     var optionsMenu: Menu? = null
     var poiLayer: PoiLayer? = null
     var destination: Feature? = null
+    var path: PathLayer? = null;
```

Comment:
✔ - killed the rest of them as well. really want to figure out how to write a checkstyle now